app/controllers/groups_controller.rb

```diff
@@ -3,7 +3,7 @@ class GroupsController < ManageCmrController
   include GroupsHelper
   include GroupEndpoints
   include PermissionManagement
-
+  include Cmr
```

Comment:
Yeah, it didn't work without it.

Reply:
that seems very odd, because if you look through the controller, `cmr_client` is in most of the actions, yet we hadn't had to include it until this change. Can you try again? I will take a look also.